### PR TITLE
1st code quality review

### PR DESCRIPTION
**Pull request by nparmar1**

### CODE REVIEW

File: baconNumber.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    return actorGraph[actor];
+};
+
+const getBaconNumber = (initialActor) => {
+    const queue = new Queue();
+    const visited = new Set();
+
+    visited.add(initialActor);
+    queue.enqueue({ actor: initialActor, pathSofar: 0 });
```
Comment (Shreshth3) [inline baconNumber.js:65]: `pathSofar` sounds like it stores a path. I'd assume that this an array of each actor name we have seen in the current path (e.g. `['Tom', 'George', ...]`). Maybe name this `pathLengthSofar`

File: baconNumber.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    Monica: ['Grace', 'Taylor'],
+    Taylor: ['Monica', 'Robert'],
+    Robert: ['Taylor', 'Hayley'],
+    Hayley: ['Robert', 'Jessica'],
+    Jessica: ['Hayley', 'Jennifer'],
+    Jennifer: ['Jessica', 'Kate'],
+    Kate: ['Jennifer'],
+};
+
+const targetActor = 'Kevin Bacon';
```
Comment (Shreshth3) [inline baconNumber.js:51]: This is a constant value, it never changes. So, it should be named in all caps, e.g. `TARGET_ACTOR`

File: baconNumber.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    const visited = new Set();
+
+    visited.add(initialActor);
+    queue.enqueue({ actor: initialActor, pathSofar: 0 });
+
+    while (queue.size() > 0) {
+        const { actor, pathSofar } = queue.dequeue();
+
+        // Process node
+        const isKevin = actor === targetActor;
```
Comment (Shreshth3) [inline baconNumber.js:71]: This is a bit misleading. The variable is named `isKevin` but the check says `actor === targetActor`. In fact, your check is validating that the actor is the target actor (and in this case, the target actor happens to be "Kevin Bacon"). You want your code to work generically (so even in the case where the target actor is not Kevin Bacon). So, you should name things generically. In this case, I'd say `isTargetActor`.

File: shortestPath.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+    return graph;
+};
+
+const getShortestPathLength = (edges, startNode, targetNode) => {
+    const queue = new Queue();
+    const visited = new Set();
+
+    visited.add(startNode);
+    queue.enqueue({ node: startNode, pathFromStart: 0 });
```
Comment (Shreshth3) [inline shortestPath.js:33]: `pathLengthFromStart`

**Review by Shreshth3 — CHANGES_REQUESTED** [review]

File: baconNumber.js (hunk 1 above)
Comment (nparmar1) [reply baconNumber.js]: Changes have been made.

File: baconNumber.js (hunk 2 above)
Comment (nparmar1) [reply baconNumber.js]: Changes have been made.

File: baconNumber.js (hunk 3 above)
Comment (nparmar1) [reply baconNumber.js]: Changes have been made.

File: shortestPath.js (hunk 4 above)
Comment (nparmar1) [reply shortestPath.js]: Changes have been made.

**Review by Shreshth3 — APPROVED** [review]

**Pull request merged**